examples/README.md

```diff
@@ -0,0 +1,21 @@
+# Simple example running on two separate docker containers
+
+## Build Docker Images
+
+The `Dockerfile` contains one `base` image and two targets: `client` and `server`
+
+```sh
+docker build --target server -t das-node-test-server --load -f examples/Dockerfile .
```

Comment:
Please test any command line you write in user's documentation by copying-and-pasting to a terminal before submitting it because copy-and-paste is exactly with everyone will do when they see a command line snippet like this.

There's a typo here and in the next line which are with wrong docker file names.

Reply:
I did copied and run, and it seems to work on my machine.
Can't see the typo anywhere.